source/Camera/Basic/CaptureTutorial.md

```diff
@@ -158,11 +158,11 @@ settings2D.set(Zivid::Settings2D::Brightness{ 1.0 });
 
 Zivid Studio can store the current settings to .yml files. These can be read and applied in the API. You may find it easier to modify the settings in these (human-readable) yaml-files in your preferred editor.
 ```cpp
-camera.setSettings(Zivid::Settings("frame_01.yml"));
+camera.setSettings(Zivid::Settings("Frame01.yml"));
```

Comment:
Has the case been aligned with @apartridge? It won't be the same, but a good opportunity to check what's in store for next SDK.

Reply:
There is no formal format agreed in the SDK. I say we accept this one and keep it going forward.

Reply:
@apartridge, do you select file name in Studio? If so, can you use CamelCase with capital first letter?